src/main/java/seedu/address/model/person/Importance.java

```diff
@@ -18,6 +18,8 @@ public class Importance {
      */
     public static final String VALIDATION_REGEX = "[^\\s].*";
 
+    private static final String[] VALID_IMPORTANCE_VALUES = {"low", "medium", "high"};
```

Comment:
Good that you are creating an array of valid values. Could consider using an ENUM?